seedlet/assets/sass/blocks/search/_style.scss

```diff
@@ -1,21 +1,58 @@
 .wp-block-search {
-    display: flex;
-    max-width: var(--responsive--aligndefault-width);
-    .wp-block-search__label {
-        font-weight: normal;
+    flex-wrap: wrap;
+    &.wp-block-search__button-inside {
+		.wp-block-search__inside-wrapper{
+            background: transparent;
+            border: var(--form--border-width) solid var(--form--border-color);
+            border-radius: var(--form--border-radius);
+            .wp-block-search__input {
+                padding: var(--form--spacing-unit);
+            }
+            .wp-block-search__button {
+                padding: var(--button--padding-vertical) var(--button--padding-horizontal);
+                &.has-icon {
+                    padding: 0;
+                }
+            }
+        }
     }
+
+	.wp-block-search__label {
+		font-weight: normal;
+	}
+
     .wp-block-search__input {
+        font-family: var(--form--font-family);
+        font-size: var(--form--font-size);
+        line-height: var(--form--line-height);
         border: var(--form--border-width) solid var(--form--border-color);
         border-radius: var(--form--border-radius);
         color: var(--form--color-text);
         line-height: var(--global--line-height-body);
         max-width: inherit;
         margin-right: calc( .66 * var(--global--spacing-horizontal) );
         padding: var(--form--spacing-unit);
+        background: transparent;
 
         &:focus {
             color: var(--form--color-text);
             border-color: var(--form--border-color);
         }
     }
+	.wp-block-search__button {
+        @extend %button-style;
```

Comment:
I think your editor has gone back to spaces here. We can also do this with a mixin rather than an extend.

Reply:
this was done like this by seedlet before, I can change that of course

Reply:
Yeah I added the mixin to Seedlet, but I didn't change every use of the `extends` but we should change them as we encounter them.